website/authoring-assistant/fluid/aggregation.fld

```diff
@@ -0,0 +1,12 @@
+let sumEmissions year data =
```

Comment:
This code needs a bit of refactoring/cleanup -- why the `totals` local var, why `records` twice`, why have `year` as a variable but `2015` as a literal, etc?